models/Meme.js

```diff
@@ -0,0 +1,37 @@
+import mongoose, { Schema } from 'mongoose';
+
+const Meme = new Schema({
+  id: Schema.ObjectId,
+  topText: {
+    type: String,
+  },
+  bottomText: {
+    type: String,
+  },
+  image: {
+    type: String,
+    required: 'Please select a meme image first'
+  },
+  name: {
+    type: String,
+    required: 'A meme must have a name'
+  },
+  userId: {
```

Comment:
We could try this also?
```suggestion
  user: {
    type: Schema.Types.ObjectId,
    ref: 'User'
  },
```